packages/documentation/src/pages/getting-started/internal-tools.mdx

```diff
@@ -1,153 +1,134 @@
-1. Copy down the SSH configuration that you'll need locally to access the remote SSH servers.
- 	  1. Visit https://github.com/department-of-veterans-affairs/devops/blob/master/ssh/config
- 	  1. Click the "Raw" button, and copy the URL that's in your browser.
- 	  1. Run `curl URL > ~/.ssh/config`, where URL is the URL that you copied in the previous step. This will overwrite any existing ssh config without warning; please back up any existing file if it exists.
-1. Add your SSH key to your local agent with `ssh-add -K ~/.ssh/id_rsa_vetsgov`.
-1. If you're on OSX you can configure the ssh client to automatically add your keys with the following in `~/.ssh/config`:
-    ```
-      Host *
-        AddKeysToAgent yes
-        UseKeychain yes
-        IdentityFile ~/.ssh/id_rsa
-        IdentityFile ~/.ssh/id_rsa_vetsgov
-    ```
-1. Once your PR is merged, an update job should automatically run to update the jumpboxes and SOCKS proxy endpoints. This job takes about 5 minutes to run. Wait five minutes, then proceed to the instructions below about accessing the SOCKS proxy.
-    * If your key doesn't seem to be working, contact the Platform team in the *#vetsgov-devops* Slack channel.
+## Prerequisites and assumptions
+
+- These steps assume you're running on Linux or OSX.
+- That you have a Github account that's [configured](https://help.github.com/en/articles/adding-a-new-ssh-key-to-your-github-account) to use `.ssh` authentication with one of your computer's public keys.
+
+## Create an SSH public key
+
+*You can skip these steps if you have a public ssh key you want to use. These instructions refer to the public key as `id_rsa_vagov.pub`*
+
+1. Open a terminal and create an `~/.ssh` directory
+```bash
+mkdir ~/.ssh
+```
+*It's safe to ignore the error if this directory already exists*
+
+2. Set the `~/.ssh` directory to read only.
+```bash
+chmod 700 ~/.ssh
+```
+
+3. Change to the `~/.ssh` directory
+
+```bash
+cd ~/.ssh
+```
+
+4. Generate the ssh keys. You will be prompted to enter a secure passphrase to protect your private key.
+
+```bash
+ssh-keygen -f id_rsa_vagov
+```
+5. Confirm that the keys were generated.
 
+```bash
+ls ~/.ssh
+```
+*The list of files should include `id_rsa_vagov` and `id_rsa_vagov.pub`. Only share keys with `.pub` (public) extension; Private keys should never leave your computer.*
 
-#### Configure the SOCKS proxy - for external developers
+## Configure the SOCKS proxy
+Below are configuration instructions for internal and external developers.
 
-1. As part of onboarding, you should have provided your **public SSH key** to DSVA. If you haven't, do that now.
-2. DSVA will add your public SSH key to the list of authorized keys.
-3. When that process is completed (24 hours), you will receive a Github notification that your key has been added. Once you receive the notification, continue with the steps below.
-    * If your key doesn't seem to be working, ask for help in the *#devsupport-external* Slack channel
```

Comment:
As much as possible I tried to retain the important information when editing. Like this was moved up to line 50 and combined with a part that was more relevant.